Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BoxViewportEdit.h

```diff
@@ -16,8 +16,9 @@ namespace AzToolsFramework
 {
     class LinearManipulator;
 
-    /// Wraps 6 linear manipulators, providing a viewport experience for 
-    /// modifying the extents of a box
+    /// Wraps 6 linear manipulators, providing a viewport experience for modifying the extents of a box.
```

Comment:
nit: I think this top line should be //! too

Reply:
The updated comment is super helpful 👍

Reply:
Fixed the //! style